Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Builder image
+FROM alpine:3.10 AS builder
+
+RUN apk add --no-cache --update \
+        autoconf \
+        automake \
+        build-base \
+        git \
+        libaio-dev \
+        libtool \
+        pkgconf
+
+RUN git clone https://github.com/akopytov/sysbench.git
```

Comment:
Can we please use an explicit version (better yet, a released one from the stable branch)?

Reply:
Done